web/themes/custom/votegov/templates/layout/html.html.twig

```diff
@@ -74,12 +74,14 @@
     </script>
     {% if not logged_in %}
       <!-- Google Tag Manager -->
+      <!--
```

Comment:
Did you read any where that this would need to be commented out?  we will need Goggle Tag Manger to help with data tracking and information